includes/utils.inc

```diff
@@ -926,22 +926,22 @@ function wf_crm_get_fields($var = 'fields') {
         'name' => ts('Contribution Page'),
         'type' => 'hidden',
         'expose_list' => TRUE,
-        'empty_option' => t('None'),
+        'empty_option' => (string) t('None'),
```

Comment:
All of these instances of casting to a string right away aren't ideal:

> "This object, when cast to a string, will return the formatted, translated string. Avoid casting it to a string yourself, because it is preferable to let the rendering system do the cast as late as possible in the rendering process, so that this object itself can be put, untranslated, into render caches and thus the cache can be shared between different language contexts."

See https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21StringTranslation%21TranslatableMarkup.php/class/TranslatableMarkup/8.6.x 

Could these be handled better later on, rather than cast to a string?

Or, if that's a whole project in itself, maybe we could make an issue to handle this later and then put `// @todo` with references to this issue everywhere that we're casting translatable markup to a string?

Reply:
We can't, due to the way Webform dumps to YAML. Any objects explode it. There's somewhere else in the code that walks an object to convert all translatable markup to strings.

Any string in these function, which is translatable, gets dumped into Webform YAML anyways. 

If we remove that, the form dies due to Yaml encoder not being setup to handle objects (I wish it just did the casting for us.)

Reply:
Er, ok, so if the untranslated strings are meant to be in Yaml, and something in Webform turns them into translateable strings, then we'll want to NOT use translated strings here. Otherwise, the content will end up being stuck in some language and no longer translateable.

For example, let's say the current user is using the UI is Polish, so `(string) t('None')` becomes the string `'Nic'`. Then `'Nic'` gets saved in the Yaml, and whatever process walks that and makes translateable strings will do the equivalent of `t('Nic')` which won't find anything, and the text is now stuck now in Polish.